docker-in-action.md

```diff
@@ -113,20 +113,20 @@ __Containers should be started in reverse order of their dependency.__ Like a DA
 
 Software needs to be system agnostic to be runnable anywhere.
 
-There are three specific features to help build environemnt-agnostic systems:
+There are three specific features to help build environment-agnostic systems:
 
 - Read-only file system
 - Environment variable injection
 - volumes
 
-For example if we are running a wordpress container. Wordpress requires a mysql db. So a wordpress has a read-only file system
+For example if we are running a wordpress container. Wordpress requires a mysql db. So wordpress has a read-only file system
 
 
 ### Read-only file systems.
 
-Read-only file systems ar ebeneficial in two ways.
+Read-only file systems ar beneficial in two ways.
 
-- Tere is confidence that the container won't be specialized from changes to the files it contains.
+- There is confidence that the container won't be specialized from changes to the files it contains.
 - An attacker can't compromise files in the container.
 
 
```

Comment:
Endeavour of below example (Wordpress containers) is a bit unclear. What are we trying to show by this?